opentelemetry-otlp/src/lib.rs

```diff
@@ -197,6 +198,20 @@ pub struct OtlpPipelineBuilder {
     trace_config: Option<sdk::trace::Config>,
 }
 
+/// Target to which the exporter is going to send spans or metrics, defaults to https://localhost:4317.
+const OTEL_EXPORTER_OTLP_ENDPOINT: &str = "OTEL_EXPORTER_OTLP_ENDPOINT";
```

Comment:
What about other parts of SDK:
https://github.com/open-telemetry/opentelemetry-rust/blob/main/opentelemetry/src/sdk/trace/span_processor.rs#L52

I think having similar constants across SDK is nice.

Reply:
I think they too, are pointless (but I just recently got involved). What do others think?

Reply:
I think as long as we keep the name convention consistent across the sdk it should be fine. Those constant are private so we don't really need the full name.

Reply:
@TommyCpp not sure what you mean by keeping the name convention consistent? Are you saying that you think having constants which contain their exact name is still useful?

Reply:
@djc Not really. It's just we have lots of environment variables const in different modules/crates. It would be wired that some of them use full name and some of them are not. So I think we should keep the name convention of those const consistent.

Reply:
It is more useful when the value is used in multiple places as it is easy to have subtle bugs if there are typos in these values. More of a tossup for single use constants, but I think the convention for env vars should probably be assessed in a subsequent PR if we want to make improvements there.

Reply:
Fair enough.